source/MaterialXTest/ShaderValid.cpp

```diff
@@ -289,7 +289,7 @@ static mx::GlslValidatorPtr createGLSLValidator(bool& orthographicView, const st
         std::string geometryFile;
         if (fileName.length())
         {
-            geometryFile =  mx::FilePath::getCurrentPath().asString() + "/documents/TestSuite/Geometry/" + fileName;
+            geometryFile =  mx::FilePath::getCurrentPath().asString() + "documents/TestSuite/Geometry/" + fileName;
```

Comment:
Why not use mx::FilePath::getCurrentPath() / mx::FilePath("documents/TestSuite/Geometry") / mx::FilePath(filename); ?